Packs/TwitterIOCHunter-FullDailyFeed/Integrations/TwitterIOCHunterFullDailyFeed/TwitterIOCHunterFullDailyFeed.yml

```diff
@@ -83,8 +83,24 @@ configuration:
   - WHITE
   required: false
   type: 15
+- additionalinfo: Select type of feed to get. Full for all, Username for filter by
+    username and Hashtag for filtered by the hashtag.
+  defaultvalue: Full
+  display: Type of Feed
+  name: typeoffeed
+  options:
+  - Full
+  - Username
+  - Hashtag
+  required: false
+  type: 15
+- additionalinfo: Input the Username or Hashtag you want to use as a filter
+  display: Filter to Use
+  name: filtertouse
+  required: false
+  type: 0
 description: Fetch the full daily feed from www.tweettioc.com/v1/tweets/daily/full
-display: TwitterIOCHunter - Full Daily Feed
+display: TwitterIOCHunter Feed
```

Comment:
Any reason for name change?

Reply:
Reason for change is that now you can filter the full list with username or hashtag -> its not the Full Daily anymore if you filter.

Reply:
Yes, that's my idea
After adding the filters to the feed it is no longer just a Full Daily Feed

Reply:
Great. Merging